typescript/src/lib/moderation/managers/ModerationManager.ts

```diff
@@ -122,12 +122,21 @@ export class ModerationManager extends Collection<number, ModerationEntity> {
 		return this;
 	}
 
-	public async count() {
-		if (this._count === null) {
-			const { moderations } = this.db;
-			this._count = await moderations.count({ where: { guildID: this.guild.id } });
-		}
-		return this._count!;
+	public async getCurrentModerationID() {
```

Comment:
No need to prefix ID as a moderation one, inside a moderation manager.
```suggestion
	public async getCurrentID() {
```

Reply:
Also rename all instances calling this.